src/app/admin/admin-push-notification-send.ts

```diff
@@ -3,7 +3,7 @@ import { AccountsRepository } from "@services/mongoose/accounts"
 import { UsersRepository } from "@services/mongoose/users"
 import { NotificationsService } from "@services/notifications"
 
-export const sendAdminPushNotification = async ({
+export const adminPushNotificationSend = async ({
```

Comment:
our current convention:

graphql layer --> verb at the end
app layer --> verb at the beginning